### ISSUE TITLE
Blocky clouds, vanilla sun, and vanilla moon not seen in reflections

### ISSUE THREAD

**Issue by andrwcrr** [sixthsurge/photon#73]
### What happened?

When `Blocky Clouds`, `Vanilla Sun`, and `Vanilla Moon` are enabled their reflections are not seen but rather the reflections of the default realistic clouds and circular sun and moon. Expected behaviour is the blocky clouds and vanilla sun and moon to show in the reflections instead. Example:
![2023-11-06_20 05 21](https://github.com/sixthsurge/photon/assets/128940455/90cf09e6-d211-4609-846d-a7d7acc5c7dc)

![2023-11-06_20 05 44](https://github.com/sixthsurge/photon/assets/128940455/8023b321-82fd-4de4-bab4-2852584fa81b)

### Minecraft Version

1.20.1

### Which shader mod are you using?

Iris

### Shader mod version

1.6.10

### Shader settings

```conf
#Mon Nov 06 20:05:04 AST 2023
WATER_DISPLACEMENT=false
WATER_WAVES=false
VANILLA_SUN=true
BLOCKY_CLOUDS=true
```

**Comment by Patomosm** [comment]
I confirm the issue only when the user doesn't play with custom texture packs. In that case reflected clouds are the ones of "Daily Weather System", not the vanilla ones.
Otherwise, the reflection is too blurry to distinguish realistic clouds from blocky photon ones.

**Comment by tsumi7** [comment]
It occurs with a resource pack too. It may not be as visible with waves enabled but it's still there.

**Comment by Patomosm** [comment]
Yes, I meant that to get the blocky clouds like in the first image, you need to have a package that doesn't have a custom-sky.

**Comment by Phryzen** [comment]
This is still an issue smh
